File: Day Five/Python/card_validator.py
def card_validator(card_digits):
	card_digits = card_digits.replace('-', '')
	if len(card_digits) != 16:
		return "Invalid card number"
	elif card_digits[0] != '4' and card_digits[0] != '5' and card_digits[0] != '6' :
		return "Invalid card number"
	else:
		return "Valid"



print(card_validator("5234-5678-9012-3456"))

File: Day Five/JS/cardValidator.js
"use strict"
function cardValidator(cardDigits){
	cardDigits = cardDigits.replace('-', '')
	if (cardDigits.length !== 16) {
        	return "Invalid card number";
   	} else if (cardDigits[0] !== '4' && cardDigits[0] !== '5' && cardDigits[0] !== '6') {
        	return "Invalid card number";
    	} else {
        	return "Valid";
    	}
}
console.log(cardValidator("4234-5678-9012-3456"))